src/file_lists.cmake

```diff
@@ -254,6 +254,8 @@ set(libprotobuf_lite_hdrs
   ${protobuf_SOURCE_DIR}/src/google/protobuf/metadata_lite.h
   ${protobuf_SOURCE_DIR}/src/google/protobuf/parse_context.h
   ${protobuf_SOURCE_DIR}/src/google/protobuf/port.h
+  ${protobuf_SOURCE_DIR}/src/google/protobuf/port_def.inc
```

Comment:
Any idea why these weren't included before (or how this worked w/o them)?

Reply:
It looks like the :port_def target was included in the `cc_dist_library` for libprotobuf but not the one for libprotobuf-lite. I'm guessing the reason we didn't notice is that the build still probably worked even without the undeclared dependency (i.e. the C++ compiler would still see `port_def.inc` in the include path even though CMake didn't know about it).